models/instrument.py

```diff
@@ -2190,7 +2190,7 @@ def exposure_depth( self, index ):
         Parameters
         ----------
           index: int
-            Index into encapsulated expolsures
+            Index into encapsulated exposures
```

Comment:
this typo ("expolsures") still pops up 4 times, and  "implemetned" pops up 3 times in this block of similar nearby code blocks in this file. Not critical, but you could search for those if it bothers you.